services/autoscaling/src/simcore_service_autoscaling/core/settings.py

```diff
@@ -111,9 +116,22 @@ class EC2InstancesSettings(BaseCustomSettings):
         "a tag must have a key and an optional value. see [https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html]",
     )
 
+    @validator("EC2_INSTANCES_TIME_BEFORE_DRAINING")
```

Comment:
since you truncating, why don't you do it as `pre`-validator?.

Reply:
well I want to have the correct time. if someone is putting ##@$ then I prefer pydantic to handle it for me.